package.json

```diff
@@ -24,8 +25,9 @@
     "eslint-plugin-eslint-plugin": "^2.2.1",
     "eslint-plugin-mocha": "^6.2.2",
     "fs-extra": "^8.1.0",
-    "husky": "^3.1.0",
+    "husky": "^4.2.1",
```

Comment:
I think this is no longer needed? See https://github.com/strongloop/loopback-next/pull/4639/files